ci/sendmail.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python3
+from email.mime.base import MIMEBase
+import email
+import ci.util
+import glci.notify
+import glci.util
+import os
+from string import Template
+
+
+body = '''
+<!DOCTYPE html><html>
+<head>
+<meta http-equiv=3D"Content-Type" content=3D"text/html; charset=3Dutf-8">
+</head>
+<body>
+<p> First paragraph </p>
```

Comment:
i think this is some debugging / test code..

Reply:
removed